package.json

```
@@ -46,10 +46,14 @@
     "clsx": "^2.1.1",
     "cmdk": "1.0.0",
 
+    "framer-motion": "^12.4.7",
```

Biome (1.9.4)

[error] The key framer-motion was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)